### PR TITLE
v1.2.0 release blog post

### PR DESCRIPTION
**Pull request by robmarcer**

### CODE REVIEW

File: src/blog/2022/12/flowforge-1-2-0-released.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
+---
+title: FlowForge 1.2 is now available with support for single sign on and persistant context storage
```
Comment (ZJvandeWeg) [inline src/blog/2022/12/flowforge-1-2-0-released.md:2]:
```suggestion
title: FlowForge 1.2 is now available with single sign on and persistent context storage
```

File: src/blog/2022/12/flowforge-1-2-0-released.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+We're pleased to announce version 1.2 is now available! The latest release of the FlowForge application contains new features, improvements, better documentation, and bug fixes.
+
+We've put a great deal of work in this release to make it easier to run your own self-managed instance of FlowForge. That includes significant improvements for FlowForge in [Kubernetes](https://flowforge.com/docs/install/kubernetes/) and [Docker](https://flowforge.com/docs/install/docker/).
+
+Keep reading for the details of what's in this release or you can watch our 1 minute roundup video of the new release above.
+
+## Features
+
+- [Single Sign-On](https://github.com/flowforge/flowforge/issues/226) Single sign-on (SSO) is a method of authentication that allows a user to access multiple applications or systems with a single set of login credentials, improving security, productivity, and user experience, and reducing IT overhead. We've implemented SSO using the Security Assertion Markup Language [(SAML)](https://en.wikipedia.org/wiki/Security_Assertion_Markup_Language) framework. This allows users of FlowForge Cloud, Enterprise and Community Edition to easily access their FlowForge projects.
```
Comment (ZJvandeWeg) [inline src/blog/2022/12/flowforge-1-2-0-released.md:22]:
```suggestion
[Single Sign-On](https://github.com/flowforge/flowforge/issues/226) Single sign-on (SSO) is a method of authentication that allows a user to access multiple applications or systems with a single set of login credentials, improving security, productivity, and user experience, and reducing IT overhead. We've implemented SSO using the Security Assertion Markup Language [(SAML)](https://en.wikipedia.org/wiki/Security_Assertion_Markup_Language) framework. This allows users of FlowForge Cloud, Premium and the open source edition to easily access their FlowForge projects.
```

File: src/blog/2022/12/flowforge-1-2-0-released.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+- [Persistent Context](https://github.com/flowforge/flowforge/issues/212) Node-RED provides a way to store information that can be shared between different nodes without using the messages that pass through a flow. This is called ‘context’. You can now select if context should be stored in memory or in persistent storage. Persistent storage allows the stored values to be recalled after a restart of your project.
```
Comment (ZJvandeWeg) [inline src/blog/2022/12/flowforge-1-2-0-released.md:24]:
```suggestion
[Persistent Context](https://github.com/flowforge/flowforge/issues/212) Node-RED provides a way to store information that can be shared between different nodes and flow executions without using the messages that pass through a flow. This is called ‘context’. You can now select if context should be stored in memory or in persistent storage. Persistent storage allows the stored values to be recalled after a restart of your project.
```

**Review by ZJvandeWeg — CHANGES_REQUESTED** [review]

File: src/blog/2022/12/flowforge-1-2-0-released.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+
+## Improvements
+
+- In FlowForge 1.1 we added logging of user actions. In 1.2 we’ve [improved the audit log interface](https://github.com/flowforge/flowforge/issues/517) to help you read the recorded user actions.
+
+- We've added documentation explaining how to deal with [DNS records in Docker and Kubernetes](https://github.com/orgs/flowforge/projects/1/views/33).
```
Comment (ZJvandeWeg) [inline src/blog/2022/12/flowforge-1-2-0-released.md:30]:
```suggestion
Configuring DNS for FlowForge has historically been challenging as for most FlowForge installs you'll need two entries. One for the FlowForge application, and one for the Node-RED projects. There's been updates to the documentation to make it much easier to set this up, and much faster. Please checkout the new [documentation](<TODO link to docs>).
```

File: src/blog/2022/12/flowforge-1-2-0-released.md [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+
+- We've updated our documentation to always link to the latest build (older builds are still available).
+
+- We are now pushing our Docker builds to Docker Hub, this saves users from having to build the Docker images when installing or updating. We've [updated our documentation](https://github.com/flowforge/flowforge/commit/c9363c8d422569df8c0a25767f9e4a647ca032ee) to reflect this improvement.
```
Comment (ZJvandeWeg) [inline src/blog/2022/12/flowforge-1-2-0-released.md:34]:
```suggestion
Previously customers were asked to build their own containers for the main FlowForge applications, as well as the Node-RED ones. For the Node-RED containers this allows customers to pre-install packages in the container they intent to use. For FlowForge Cloud these containers are build by FlowForge. These containers are now published to the [Docker Hub](https://hub.docker.com/u/flowforge). This makes it much easier to get up and running with your first containers.

We are now pushing our Docker builds to Docker Hub, this saves users from having to build the Docker images when installing or updating. These containers are used by default by `docker-compose`.
```

File: src/blog/2022/12/flowforge-1-2-0-released.md [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+
+- We've improved the documentation around configuration of [MQTT](https://github.com/flowforge/flowforge/issues/1397).
```
Comment (ZJvandeWeg) [inline src/blog/2022/12/flowforge-1-2-0-released.md:36]:
```suggestion
Setting up MQTT for inter-project communication and communication with devices has been simplified. Please read the improved the documentation around configuration of [MQTT](https://github.com/flowforge/flowforge/issues/1397).
```

File: src/blog/2022/12/flowforge-1-2-0-released.md (hunk 2 above)
Comment (ZJvandeWeg) [reply src/blog/2022/12/flowforge-1-2-0-released.md:22]: @robmarcer Please tell people to create a support ticket through the contact us page, or support@flowforge.com to enable this for their team on FlowForge cloud. Also link to the admin docs for Premium customers.

Comment (ZJvandeWeg) [reply src/blog/2022/12/flowforge-1-2-0-released.md:22]: See also: https://github.com/flowforge/flowforge/blob/main/docs/cloud/README.md#single-sign-on

File: src/blog/2022/12/flowforge-1-2-0-released.md [hunk 7]
```diff
@@ -0,0 +1,85 @@
...
+
+## Bug Fixes
+
+We've fixed the following bugs in this release.
+
+- Unable to edit 'Prevent Install of External nodes' template option [#1376](https://github.com/flowforge/flowforge/issues/1376)
```
[the remark this answers is not in the document]
Comment (robmarcer) [reply src/blog/2022/12/flowforge-1-2-0-released.md:42]: Yes, last cycle we also mentioned stuff we'd done in minor releases.

**Comment by robmarcer** [comment]
@ZJvandeWeg everything should now be ready to go, can you review please?

File: src/blog/2022/12/flowforge-1-2-0-released.md [hunk 8]
```diff
@@ -0,0 +1,85 @@
...
+
+- Email features do not pick up if email server is offline when FF launches [#1159](https://github.com/flowforge/flowforge/issues/1159)
```
[the remark this answers is not in the document]
Comment (robmarcer) [reply src/blog/2022/12/flowforge-1-2-0-released.md]: I've updated this text.

File: src/blog/2022/12/flowforge-1-2-0-released.md [hunk 9]
```diff
@@ -0,0 +1,85 @@
...
+
+- [Persistent Context](https://github.com/flowforge/flowforge/issues/212) Node-RED provides a way to store information that can be shared between different nodes without using the messages that pass through a flow. This is called ‘context’. You can now select if context should be stored in memory or in persistent storage. Persistent storage allows the stored values to be recalled after a restart of your project.
+
+## Improvements
+
+- In FlowForge 1.1 we added logging of user actions. In 1.2 we’ve [improved the audit log interface](https://github.com/flowforge/flowforge/issues/517) to help you read the recorded user actions.
```
[the remark this answers is not in the document]
Comment (robmarcer) [reply src/blog/2022/12/flowforge-1-2-0-released.md:28]: I've added the screenshot

**Pull request merged**